lms/djangoapps/courseware/tests/test_entrance_exam.py

```diff
@@ -206,6 +211,100 @@ def setUp(self):
             ]
         )
 
+    @mock.patch('xmodule.x_module.XModuleMixin.has_dynamic_children', mock.Mock(return_value='True'))
+    def test_view_redirect_if_entrance_exam_required(self):
```

Comment:
@cahrens -- by 'going directly' can you explain further what you are experiencing?  I've logged in as the Verified user on the sandbox and I can't seem to get around the entrance exam gating.  In addition, the messaging indicates that the required score is 100% and my score is currently 0%.